Wikipedia.xcodeproj/project.pbxproj

```diff
@@ -6819,6 +6825,8 @@
 				RUN_CLANG_STATIC_ANALYZER = NO;
 				SWIFT_OBJC_BRIDGING_HEADER = "Wikipedia/Code/Wikipedia-Bridging-Header.h";
 				WMF_HOCKEYAPP_IDENTIFIER = 5d80da08a6761e5c6456736af7ebad88;
```

Comment:
Hi, part (b) not seeing crash dialogue was happening on the beta; though
once I deleted and reinstalled per Joe's comment earlier the
dialogue appeared again.

Part (a) where none of the tweak settings were working was on my local.



On Thursday, 9 June 2016, Corey Floyd <notifications@github.com
<javascript:_e(%7B%7D,'cvml','notifications@github.com');>> wrote:

> In Wikipedia.xcodeproj/project.pbxproj
> <https://github.com/wikimedia/wikipedia-ios/pull/708#discussion_r66463179>
> :
>
> > @@ -6819,6 +6825,8 @@
> >  				RUN_CLANG_STATIC_ANALYZER = NO;
> >  				SWIFT_OBJC_BRIDGING_HEADER = "Wikipedia/Code/Wikipedia-Bridging-Header.h";
> >  				WMF_HOCKEYAPP_IDENTIFIER = 5d80da08a6761e5c6456736af7ebad88;
>
> 🤕
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/wikimedia/wikipedia-ios/pull/708/files/698aeb8d54d02072a87287b267bf9e52f975ec33#r66463179>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AG59Z_bt1Etrz9bUF8iIn2QUqUTVUffQks5qKDDqgaJpZM4Iw-Ws>
> .
>